src/scss/_mixins.scss

```
@@ -0,0 +1,61 @@
+/// Output All oComments Features
+@mixin oComments ($class: 'o-comments') {
```

```suggestion
@mixin oComments () {
```

The class argument is never used and we technically can't remove it later without it being a breaking change